FiveCalls/FiveCalls/IssueDetailViewController.swift

```diff
@@ -196,7 +196,7 @@ extension IssueDetailViewController : EditLocationViewControllerDelegate {
     }
     
     func editLocationViewController(_ vc: EditLocationViewController, didUpdateLocation location: UserLocation) {
-        issuesManager.fetchIssues(.top(location)) { result in
+        issuesManager.fetchIssues(false, location: location) { result in
```

Comment:
I think I like the enum style better, but I see the need to extract the location and pass that regardless.  Maybe just remove the associated type and use the enum like this?

```swift
fetchIssues(.all, location: location)  // or .top
```